### PR TITLE
Mark implementations of num-traits actually as `pub`

### PR DESCRIPTION
**Pull request by regexident**
… as they should have been from the let go.

After all, what's the point of implementing all this stuff if you're then not making it accessible? 🤦🏻‍♂️

### CODE REVIEW

**Review by japaric — APPROVED** [review]
LGTM. Will r+ after #10 lands.

File: src/lib.rs [hunk 1]
```diff
@@ -102,6 +102,8 @@ use typenum::{Cmp, Greater, Less, U0, U1, U2, U3, U4, U5, U6, U7, U8, U9, U10,
 
 mod num_traits_impl;
 
+pub use num_traits_impl::*;
```
Comment (japaric) [inline src/lib.rs:105]: It may be clearer to replace this with `pub use num_traits_impl::ParseFixedError` as that's the only public item in that module.

Comment (regexident) [reply src/lib.rs:105]: I found that with just `use` the trait implementation were not accessible from a third-party crate. Changing to `pub use` fixed things for me. Weird, but I couldn't find a better solution.

**Pull request closed without merging**